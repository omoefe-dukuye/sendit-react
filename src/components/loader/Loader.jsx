import React from 'react';
import './styles.scss';

export default () => (
  <div className='loaderBg'>
    <div className='loader'>
    </div>
  </div>
);
