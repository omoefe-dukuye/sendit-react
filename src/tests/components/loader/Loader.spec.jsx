import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../../../components/loader';

test('Should render static component', () => {
  const wrapper = shallow(<Loader />);
  expect(wrapper).toMatchSnapshot();
});
