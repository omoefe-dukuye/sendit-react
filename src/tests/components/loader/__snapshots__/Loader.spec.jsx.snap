// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render static component 1`] = `
<div
  className="loaderBg"
>
  <div
    className="loader"
  />
</div>
`;
